BuildSrc/manifestReplacementTest/buildSrc/src/main/kotlin/VerifyManifestTask.kt

```diff
@@ -13,34 +13,29 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import com.android.build.api.variant.BuiltArtifactsLoader
+import java.io.File
 import org.gradle.api.DefaultTask
 import org.gradle.api.file.DirectoryProperty
-import org.gradle.api.tasks.InputFiles
-import org.gradle.api.tasks.TaskAction
-import java.io.ByteArrayOutputStream
-import java.io.PrintStream
-
-import com.android.build.api.variant.BuiltArtifactsLoader
 import org.gradle.api.provider.Property
+import org.gradle.api.tasks.InputFiles
 import org.gradle.api.tasks.Internal
-import java.io.File
+import org.gradle.api.tasks.TaskAction
 
-abstract class VerifyManifestTask: DefaultTask() {
+abstract class VerifyManifestTask : DefaultTask() {
 
-    @get:InputFiles
-    abstract val apkFolder: DirectoryProperty
+    @get:InputFiles abstract val apkFolder: DirectoryProperty
 
-    @get:Internal
-    abstract val builtArtifactsLoader: Property<BuiltArtifactsLoader>
+    @get:Internal abstract val builtArtifactsLoader: Property<BuiltArtifactsLoader>
 
     @TaskAction
     fun taskAction() {
-        val builtArtifacts = builtArtifactsLoader.get().load(apkFolder.get())
-            ?: throw RuntimeException("Cannot load APKs")
-        if (builtArtifacts.elements.size != 1)
-            throw RuntimeException("Expected one APK !")
+        val builtArtifacts =
+            builtArtifactsLoader.get().load(apkFolder.get())
+                ?: throw RuntimeException("Cannot load APKs")
+        if (builtArtifacts.elements.size != 1) throw RuntimeException("Expected one APK !")
```

Comment:
can the formatter auto-add braces instead of single-lining it?
```suggestion
        if (builtArtifacts.elements.size != 1) {
            throw RuntimeException("Expected one APK !")
        }
```